setup.rst

```diff
@@ -236,7 +236,10 @@ find git for Windows on the **Individual components** tab of the installer.
   may not be configured on recent versions of Windows, such as Windows 10). If
   you are building on a recent Windows version, use the Control Panel (Programs
   | Programs and Features | Turn Windows Features on or off) and ensure that the
-  entry ".NET Framework 3.5 (includes .NET 2.0 and 3.0)" is enabled.
+  entry ".NET Framework 3.5 (includes .NET 2.0 and 3.0)" is enabled. If you are
```

Comment:
This "Note" doesn't seem like an appropriate place for this info. I'd perhaps place it at the end of this section, before/after the note about Python 2.7 using a different version of Visual Studio.